docs/changelog/118837.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 118837
+summary: Add missing timeouts to rest-api-spec ILM APIs
+area: "Cluster Coordination"
```

Comment:
Makes sense to me but these omissions pre-date #109112 and definitely belong to ILM rather than `Cluster Coordination`